components/homepage/About/About.jsx

```diff
@@ -6,6 +6,50 @@ const About = () => {
     <div id='about' className={style.about}>
       <div className={style.container}>
         <h2>Welcome to Vovoca</h2>
+ VOVOCA-5
+        <hr className="divider-main"></hr>
```

Comment:
why code is showing  VOVOCA-5 and Development. You haven't resolved conflicts properly